src/FrameReflower/AbstractFrameReflower.php

```diff
@@ -354,8 +354,8 @@ protected function _parse_content()
     {
         // Matches generated content
         $re = "/\n" .
-            "\s(counters?\\([^)]*\\))|\n" .
-            "\A(counters?\\([^)]*\\))|\n" .
+            "\s(counters?\\([^;\v}]*)|\n" .
```

Comment:
What's the deal with this change? Not quite rocking what you're doing here.

Reply:
As far as I understand the old regex would capture content starting with 'counters' which was wrapped in brackets (stopping at a closing bracket or new line). The new regex should capture everything until it hits the semi-colon, \v for vertical space or closing brace, so we can pickup on additional text to concatenate.

Reply:
Thanks, I'll go over it again and let you know if I have any feedback.